src/sp/v2.0.2/src/ncpus.F

```diff
@@ -29,6 +29,7 @@ FUNCTION NCPUS()
 C - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       INTEGER NTHREADS, TID, OMP_GET_NUM_THREADS,OMP_GET_THREAD_NUM
 C     Obtain thread number
+#if OPENMP == 1
```

Comment:
Please use `#ifdef OPENMP` instead of `#if OPENMP == 1`.

Reply:
I believe there is a problem with that solution. If the make script is run with the option -o 0, OPENMP is set to 0 in the environment (make_ncep_libs.sh, line 185). I believe this will cause the CPP directive `#ifdef OPENMP` to return 'true'.

I can modify make_ncep_libs.sh as well if you'd rather stick with this convention.

Reply:
I thought about it further, and I think my above reasoning is incorrect. If OPENMP=1 in the environment, the flag -DOPENMP is passed at compiled time due to the macros.make directives, whereas if it is any other value no flag is passed.

I still think I will modify make_ncep_libs.sh to only export OPENMP if it is equal to one, just to avoid any confusion in the future

Reply:
Sounds like the safest solution to me. Thanks for doing this. I can also add the new OPENMP logic to the other machine stanzas in a follow-up PR (and remove theia at that time).